homeassistant/components/netatmo/climate.py

```diff
@@ -241,6 +246,21 @@ async def async_added_to_hass(self) -> None:
                 )
             )
 
+        for module in self._room.modules.values():
+            if getattr(module.device_type, "value") not in [NA_THERM, NA_VALVE]:
```

Comment:
Is there a risk by making this an exclusive check? Might new device types be added in the future?

Reply:
Once the upstream lib has gotten an overhaul this will be refactored accordingly.